roles/reproducer/tasks/configure_crc.yml

```diff
@@ -32,6 +32,19 @@
         dns4_ignore_auto: true
         state: present
 
+    - name: Create dir for dnsmasq.conf
+      ansible.builtin.file:
```

Comment:
it's supposed to be a symlink pointing to /srv - do we really need this one? Care to provide some more context about the issue you're trying to solve?